hvac/utils.py

```diff
@@ -319,10 +313,8 @@ def _check_pem(arg):
     if not isinstance(param_argument, list) or not all(
         _check_pem(p) for p in param_argument
     ):
-        error_msg = (
-            "unsupported {param} public key / certificate format, required type: PEM"
-        )
-        raise exceptions.ParamValidationError(error_msg.format(param=param_name))
+        error_msg = (f"unsupported {param_name} public key / certificate format, required type: PEM")
```

Comment:
Can probably drop the parenthesis here
```suggestion
        error_msg = f"unsupported {param_name} public key / certificate format, required type: PEM"
```